app/views/courses/index.html.erb

```diff
@@ -43,7 +45,7 @@
       </tbody>
     </table>
     <% if is_admin_html %>
```

Comment:
Should a prof be able to make a course? I'm not sure on this one. Just a question.

Reply:
Ritchey, said no